imports/plugins/core/accounts/client/components/groupsTableButton.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Components, registerComponent, withPermissions } from "@reactioncommerce/reaction-components";
+
+const GroupsTableButton = ({ account, group, handleRemoveUserFromGroup, hasPermissions }) => {
```

Comment:
Should have a @jsdoc comment. example for react components: https://github.com/reactioncommerce/reaction/blob/marketplace/imports/plugins/core/ui/client/components/notFound/notFound.js#L8